h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -58,7 +58,7 @@ public AlgorithmValuesProvider() {
     // Base models
     @API(level = API.Level.critical, direction = API.Direction.INOUT,
             help = "List of models (or model ids) to ensemble/stack together. If not using blending frame, then models must have been cross-validated using nfolds > 1, and folds must be identical across models.", required = true)
```

Comment:
can you please update the `help` in annotation? This is used to generate doc for clients (including fields description in Flow).